packages/ad/package.json

```
@@ -14,6 +14,7 @@
     "test:ios": "jest --config='./__tests__/ios/jest.config.js'",
     "test:web": "jest --config='./__tests__/web/jest.config.js'",
     "test:all": "yarn test:android --coverage && yarn test:ios --coverage && yarn test:web --coverage",
+    "watch": "watch 'yarn prepublishOnly && cd ../ssr && yarn prepublishOnly' ./src --ignoreDotFiles --ignoreUnreadable",
```

My suggestion is to run `yarn bundle` in given package. It will reproduce `rnw.js` file which is our purpose to get new changes. Command `prepublishOnly` run first `yarn transpile`  - clean up and rebuild the 'dist' folder which isn`t needed in our case.
Supposed to be applied for each package.json.

```suggestion
    "watch": "watch 'yarn bundle && cd ../ssr && yarn prepublishOnly' ./src --ignoreDotFiles --ignoreUnreadable",
```